src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimSetAsideJudgmentDefendantNotificationHandler.java

```diff
@@ -73,19 +79,47 @@ DEFENDANT_NAME_INTERIM, getDefendantNameBasedOnCaseType(caseData),
 
     private CallbackResponse notifyClaimSetAsideJudgmentToDefendant(CallbackParams callbackParams) {
         CaseData caseData = callbackParams.getCaseData();
+        String recipientEmail = getRecipientEmail(callbackParams);
+        boolean isApplicantLip = caseData.isApplicantLiP();
```

Comment:
In defendant handler, why we are checking isApplicantLip ? We should be checking if respondent is Lip

Reply:
Yes, I'll fix it, thanks!